examples/triangle/main.rs

```diff
@@ -102,9 +101,10 @@ fn main() {
             }
         }
 
-        graphics.clear(clear_data, gfx::COLOR, &frame);
-        graphics.draw(&batch, &(), &frame).unwrap();
-        graphics.end_frame();
+        renderer.reset();
+        renderer.clear(clear_data, gfx::COLOR, &frame);
+        renderer.draw(&(&mesh, slice, &program, &(), &state), &frame).unwrap();
```

Comment:
I am away this weekend. So I'd bjz is happy merge away.
On Fri, Feb 13, 2015 at 9:01 PM Dzmitry Malyshau <notifications@github.com>
wrote:

> In examples/triangle/main.rs
> <https://github.com/gfx-rs/gfx-rs/pull/557#discussion_r24710741>:
>
> > @@ -102,9 +101,10 @@ fn main() {
> >              }
> >          }
> >
> > -        graphics.clear(clear_data, gfx::COLOR, &frame);
> > -        graphics.draw(&batch, &(), &frame).unwrap();
> > -        graphics.end_frame();
> > +        renderer.reset();
> > +        renderer.clear(clear_data, gfx::COLOR, &frame);
> > +        renderer.draw(&(&mesh, slice, &program, &(), &state), &frame).unwrap();
>
> I'm all for it. Not sure if @csherratt <https://github.com/csherratt> has
> finished reviewing it or not.
> How about giving him another hour or two and then merge?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gfx-rs/gfx-rs/pull/557/files#r24710741>.
>